src/encoded/types/cart.py

```diff
@@ -31,11 +36,12 @@
 class Cart(Item):
     item_type = 'cart'
     schema = load_schema('encoded:schemas/cart.json')
+    name_key = 'identifier'
 
     STATUS_ACL = {
         'current': [(Allow, 'role.owner', ['view', 'edit'])] + ALLOW_CURRENT,
-        'deleted': DELETED,
-        'disabled': ONLY_ADMIN_VIEW,
+        'deleted': [(Allow, 'role.owner', ['edit'])] + DELETED,
+        'disabled': [(Allow, 'role.owner', ['view', 'edit'])],
```

Comment:
It might be a good idea to have `+ ONLY_ADMIN_VIEW` at the end of this. We should also make sure there are permission tests for this.

Reply:
I added `+ ONLY_ADMIN_VIEW` and it seems to work properly still, and pushed this change.

Reply:
Added a couple (enough?) permissions tests for disabled carts.